spec/smoke_tests/participant_declarations_spec.rb

```diff
@@ -30,7 +25,7 @@ def fetch_smoke_test_domain
     paas_environment = ENV.fetch("ENVIRONMENT")
     YAML.load_file("#{__dir__}/../../terraform/workspace-variables/#{paas_environment}_app_env.yml")["DOMAIN"]
   rescue Errno::ENOENT
-    "ecf-#{paas_environment}.#{CLOUDAPPS_DOMAIN}"
```

Comment:
Isn't this the same thing?

Reply:
Yes, this constant has been previously defined on line 6 in the same file. I've decided to remove the constant as it was only used in this one place and I have quite strong feeling about local-but-global constants hanging around.